evap/staff/management/commands/run_tasks.py

```diff
@@ -26,12 +29,23 @@ def update_courses(self):
                     course.evaluation_end()
                     if course.is_fully_reviewed():
                         course.review_finished()
+                        if not course.is_graded:
+                            course.publish()
```

Comment:
as discussed, please make evap send mails to the admins if something goes wrong here.